src/utils.ts

```diff
@@ -14,7 +13,13 @@ export const getUniqueScopes = (...scopes: string[]) => {
 };
 
 export const parseQueryResult = (hash: string) => {
```

Comment:
I'd rename this fragment or something to avoid name conflict with the hash parameter on line 18.

Reply:
pushed